bin/runtime/src/lib.rs

```diff
@@ -895,6 +896,10 @@ impl_runtime_apis! {
                 Aleph::queued_emergency_finalizer(),
             ))
         }
+
+        fn aleph_bft_version(session: SessionIndex) -> Result<AlephBFTVersion, AlephApiError> {
+           Aleph::aleph_bft_version(session).ok_or(AlephApiError::VersionNotSet)
```

Comment:
I think what would be more suited is to get the latest set version. Lets say we set version to Legacy on the session nr `n-10` and did not change it since. Asking for the version for session `n` should return Version::Legacy. Also we need some default in case when session was never set - the legacy version seems like a perfect candidate

Reply:
Changed logic to work this way.